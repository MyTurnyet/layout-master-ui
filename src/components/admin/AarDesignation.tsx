import React, {FunctionComponent} from "react";

export type AarDesignationProps = {
    initials:string;
    description?:string;
}

const AarDesignation:FunctionComponent<AarDesignationProps>
    = (props)=>
{
    return (
        <div className={`AarDesignation`}>{props.initials}</div>
    )
}

export default AarDesignation;
