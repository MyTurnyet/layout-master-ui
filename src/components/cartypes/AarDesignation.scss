@import "../../theme/theme";

.AarDesignation {
  //justify-content: center;
  //align-content: center;
  //align-items: center;
  text-align: center;
  padding: 5px;
  font-size: 1.2em;
  width: 2.4em;
  color: $tertiary;
  background-color: $secondary-light;

}
