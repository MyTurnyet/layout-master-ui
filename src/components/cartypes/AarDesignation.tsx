import React, {FunctionComponent} from "react";

import "./AarDesignation.scss";

export interface AarDesignationProps {
    initials:string;
}

const AarDesignation:FunctionComponent<AarDesignationProps>
    = (props)=>
{
    return (
        <div className={`AarDesignation`}>{props.initials}</div>
    )
}

export default AarDesignation;
