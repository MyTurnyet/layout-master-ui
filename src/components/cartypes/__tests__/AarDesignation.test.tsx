import React from 'react';
import {render, screen} from '@testing-library/react';
import AarDesignation, {AarDesignationProps} from "../AarDesignation";
import {ThemeProvider} from "styled-components";
import defaultTheme from "../../../theme/theme";

describe(' AarDesignation ', () => {
    it('should have class and be in the document ', () => {
        const designation: AarDesignationProps = {
            initials: "GS"
        };
        render(
            <ThemeProvider theme={defaultTheme}>
                <AarDesignation {...designation} />
            </ThemeProvider>);
        const element = screen.getByText("GS");
        expect(element).toBeInTheDocument();
        expect(element).toMatchSnapshot();
    });
});
