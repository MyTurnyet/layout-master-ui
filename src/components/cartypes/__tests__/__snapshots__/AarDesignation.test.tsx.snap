// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` AarDesignation  should have class and be in the document  1`] = `
<div
  class="sc-bdfBwQ bBNLzT"
>
  GS
</div>
`;
